python/paddle/fluid/layers/detection.py

```diff
@@ -564,6 +565,48 @@ def __reshape_to_2d(var):
     return loss
 
 
+def prior_box(input,
+              image,
+              min_sizes,
+              max_sizes,
+              aspect_ratios,
+              variance,
+              flip=False,
+              clip=False,
+              step_w=0.0,
+              step_h=0.0,
```

Comment:
how about merge `step_w`, `step_h` into one `steps`?

Reply:
good idea, it will be done

Reply:
Done